lib/beaker/dsl/install_utils.rb

```diff
@@ -1,13 +1,14 @@
-[ 'foss', 'pe', 'puppet', 'ezbake', 'module' ].each do |lib|
+[ 'foss', 'puppet', 'ezbake', 'module' ].each do |lib|
     require "beaker/dsl/install_utils/#{lib}_utils"
 end
+require "beaker/dsl/install_utils/pe_defaults"
 
 module Beaker
   module DSL
     # Collection of installation methods and support
     module InstallUtils
       include DSL::InstallUtils::PuppetUtils
-      include DSL::InstallUtils::PEUtils
+      include DSL::InstallUtils::PEDefaults
```

Comment:
Is there a reason why this wasn't moved to beaker-pe? That seems like the more logical location for this set of info.